src/ext-test/java/org/opentripplanner/ext/vectortiles/VectorTilesConfigDocTest.java

```diff
@@ -0,0 +1,74 @@
+package org.opentripplanner.ext.vectortiles;
+
+import static org.opentripplanner.framework.io.FileUtils.assertFileEquals;
+import static org.opentripplanner.framework.io.FileUtils.readFile;
+import static org.opentripplanner.framework.io.FileUtils.writeFile;
+import static org.opentripplanner.framework.text.MarkdownFormatter.HEADER_4;
+import static org.opentripplanner.generate.doc.framework.DocsTestConstants.DOCS_ROOT;
+import static org.opentripplanner.generate.doc.framework.DocsTestConstants.TEMPLATE_ROOT;
+import static org.opentripplanner.generate.doc.framework.TemplateUtil.replaceSection;
+import static org.opentripplanner.standalone.config.framework.json.JsonSupport.jsonNodeFromPath;
+
+import java.io.File;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.generate.doc.framework.DocBuilder;
+import org.opentripplanner.generate.doc.framework.GeneratesDocumentation;
+import org.opentripplanner.generate.doc.framework.ParameterDetailsList;
+import org.opentripplanner.generate.doc.framework.ParameterSummaryTable;
+import org.opentripplanner.generate.doc.framework.SkipNodes;
+import org.opentripplanner.standalone.config.RouterConfig;
+import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
+import org.opentripplanner.test.support.ResourceLoader;
+
+@GeneratesDocumentation
+public class VectorTilesConfigDocTest {
```

Comment:
Btw I wonder if these doc generation tests are a rare case in tests where it would  make sense to use some interface/base class?

Reply:
I'm not going to change anything until we hear from @t2gran about that. :)

Reply:
Btw did you happen to discuss this in the dev meeting, I forgot this

Reply:
We didn't. Can you make an agenda item?